content/news/2023-02-22-news-2023-02-22-new-r-ladies-website/index.en.md

```diff
@@ -0,0 +1,103 @@
+
+In 2020, I (Athanasia Mowinckel) was on-boarded to the website team, initially to maintain the Wordpress site while the new Hugo site was being built.
+After just a little while, I also started working on the blogdown site. 
+
+In mid-2020 we decided that the webpage was better served as a pure Hugo site, not as blogdown. 
+At that point, it was due to some Hugo features not existing in Blogdown (yet) that we wanted to utilise for the website.
+These were settings for multi-lingual websites we really wanted to take advantage of.
+Furthermore, since only the blog would ever need code execution, having a site build that does not need R means a much faster build, and easier to maintain.
+
+Then Covid hit, and we all felt the stress of that period.
+Development went slow, and things dragged out.
+Thankfully, the Leadership contacted me and asked if I needed help to get the last pieces in place, and that we could set aside a little budget to hire help for the javascript pieces I was struggling with.
+
+We [announced]() the need, and hired Ben Ubah to help me get the last crucial pieces in place. Finally, we were nearing completion!
+
+## Release & new features!
+
+We have now released the new website and are thrilled about how it works so far!
+The theme is well suited to us, and the content is much easier to deal with now that we can collaborate through GitHub.
+It's also made it possible to integrate the website with certain other automatic pipelines, which enables us to have a couple of new features to the website from before!
+
+- [Events page]() with a calendar of R-Ladies events
+  - These are fetched daily from meetup through their API
+- [Direcotry page]() which is actually fast!
+  - Updated and maintained in another private repository with Airtable integration
+- [Blog]() where we welcome contributing posts and cross-posts!
+  - We'd love to see the blog revived and used by our community to show their skills and fun things they are doing with R!
+- [News page]() where the R-Ladies Global team can announce important notices about the global governance of our community
+
+And more!
+
+## Future work & call for community help
+
+There are still some things we are working on in the background, that we hope will improve the experience of our website, and also fulfil obligations we have promised previously.
+
+### Multilingual webpage
+We have set the website up to be multilingual, and have some development content for French, Spanish and Portuguese.
+But, there is still a long way to go before we get to a point where these languages are well enough translated and enough content has been translated, for the language to be released in the production website.
+Also, we are assuming that once we start having more translated content, we will see areas where the code needs fixing for proper multilingual support.
+
+We hope with community help we can make an effort to cover the major languages used by our community.
```

Comment:
what help can someone provide exactly?

Reply:
I have not updated the wiki on providng translations. But I can link to it, for sure